projects/sft-protocol/index.js

```diff
@@ -8,7 +8,7 @@ const tokens = [
 ]
 
 const getActiveMinersFromRPC = async () => {
-    const resp = await getConfig('sft-protocol', 'https://api.sftproject.io/api/v1/public/dashboard/info')
+    const resp = await getConfig('sft-protocol', 'https://ww8.sftproject.io/api/c/api/v1/public/dashboard/info')
```

Comment:
this endpoint is not reachable?

Reply:
the old endpoint is for v1, has been suspended, v2 changed to new url.